omero/sysadmins/server-upgrade.txt

```diff
@@ -131,6 +131,38 @@ below. Please refer to each section for additional details.
     :local:
     :depth: 1
 
+Check ahead for upgrade issues
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There *may* be a "precheck" SQL script provided that performs various
```

Comment:
``May`` - this is not giving me enough comfort as a user I would say. I think it would be good to tell me right here whether or not I should worry about this script (see the comment below, combine with the sentence ``If the script is not here, then you do not need to worry.``) If I get the explanation right here, I might be lucky enough to be able to skip this paragraph.